src/main/java/com/byznass/tiolktrack/kernel/util/IdentifierValidator.java

```diff
@@ -0,0 +1,12 @@
+package com.byznass.tiolktrack.kernel.util;
+
+public class IdentifierValidator {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3AS1118)